scrowl-io/scrowl
# Sc 1442 w35 glossary form

### Short summary

- Addressed QA items on ticket
- No changes made to textarea, as it seems to meet requirements
- Added break-word to text wrap for larger lines of text

---

### Test steps

1. Type large amount of text in definition field
1. Click 'save'
1. Check glossary tab to ensure that definition displays without stretching text outside visible container
